### DISCUSSION TITLE
Welcome to playwright-framework-template Discussions!

### DISCUSSION THREAD

**Discussion by abhaybharti** [abhaybharti/playwright-framework-template#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by testeranupam** [comment]
Hi Abhay your code is not working for Api test.
<img width="779" alt="Screenshot 2024-02-09 at 8 20 50 PM" src="https://github.com/abhaybharti/playwright-framework-template/assets/109193504/9b8f8c6d-af95-4490-a600-478bedf0d2bf">
